frontend/app/services/SubscriptionService.scala

```diff
@@ -155,6 +156,17 @@ class SubscriptionService(val tierPlanRateIds: Map[ProductRatePlan, String],
     } yield subscriptionDetails
   }
 
+  def getUsageCountWithinTerm(memberId: MemberId, unitOfMeasure: String): Future[Int] = (for {
+      subscriptions <- getSubscriptions(memberId)
+      details <- getSubscriptionDetails(subscriptions.head.id) if subscriptions.nonEmpty
+      startDate = formatDateTime(details.contractAcceptanceDate)
+      endDate = formatDateTime(details.effectiveStartDate + 1.year)
```

Comment:
Lose the endDate (the current term should be ongoing) - and see if you can get term start and end date out of the REST API https://knowledgecenter.zuora.com/BC_Developers/REST_API/B_REST_API_reference/Subscriptions/4_Get_subscriptions_by_account